lib/AST/ModuleDependencies.cpp

```diff
@@ -509,6 +510,86 @@ swift::dependencies::checkImportNotTautological(const ImportPath::Module moduleP
   return false;
 }
 
+void
+swift::dependencies::registerCxxInteropLibraries(
+    const llvm::Triple &Target,
+    StringRef mainModuleName,
+    bool hasStaticCxx, bool hasStaticCxxStdlib,
+    std::function<void(const LinkLibrary&)> RegistrationCallback) {
+  if (Target.isOSDarwin())
+    RegistrationCallback(LinkLibrary("c++", LibraryKind::Library));
+  else if (Target.isOSLinux())
```

Comment:
I thought Android used libc++ and not libstdc++?

Reply:
The core logic here just moved from how it was in `GenDecl.cpp`. But @egorzhdan may know the answer.